### PR TITLE
changes to the inspire-book

### PR DESCRIPTION
**Pull request by sonwabi**

### CODE REVIEW

File: css/inspire-books.css [hunk 1]
```diff
@@ -1,7 +1,11 @@
 /* inspire books */
 
 .review-stars i {
-    color: rgb(219, 152, 26);
+    color: #F1592C;
+}
+
+#star-number {
+    font-weight: bolder;
```
Comment (qodvilla) [inline css/inspire-books.css:8]: Do we have `bolder ` in CSS?

File: css/inspire-books.css [hunk 2]
```diff
@@ -74,16 +80,19 @@
...
 }
 
 .buttons-inspire .btn-download {
+    color: whitesmoke;
     border-radius: 2px;
-    background-color: #FF9F00;
+    background-color: #0076FF;
     text-align: center;
     padding: 8px 10px;
+    font-weight: bolder;
```
Comment (qodvilla) [inline css/inspire-books.css:95]: `bolder` ?

File: css/inspire-books.css (hunk 1 above)
Comment (sonwabi) [reply css/inspire-books.css:8]: yes we do

File: css/inspire-books.css (hunk 2 above)
Comment (sonwabi) [reply css/inspire-books.css:95]: yes

**Pull request merged**